src/lib/mongoBackend/mongoNotifyContext.cpp

```diff
@@ -43,7 +43,9 @@ HttpStatusCode mongoNotifyContext
   const std::vector<std::string>&  servicePathV
 )
 {
-    reqSemTake(__FUNCTION__, "ngsi10 notification");
+    bool reqSemTaken;
+
+    reqSemTake(__FUNCTION__, "ngsi10 notification", SemReadOp, &reqSemTaken);
```

Comment:
ok, I was thinking about notifying, not about getting notified ...
If processContextElement modifies the database, it should definitely be SemWriteOp.